uploader.json

```diff
@@ -277,6 +277,11 @@
             "pattern": [],
             "symbolic_name": "gluster_v_status"
         },
+	{
+            "command": "/usr/sbin/gluster volume status",
```

Comment:
This command should be `gluster peer status`.

Reply:
Also please move this entry above `gluster_v_status` so that the symbolic name is in alpha order.  Same with `uploader.v2.json`.

Reply:
@bfahr Thanks for catching the typo :+1:  The entry is moved above `gluster_v_status` as suggested.